### ISSUE TITLE
[Accessibility] Color contrast in Log in>"forgot password" Link

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4970]
# Engineers Checklist
### Describe the bug
- The "Forgot password" link does not have enough color contrast with the background, making it difficult to read. Adequate color contrast between text and background is necessary for all users, especially those with low vision or other visual impairments. 

### To Reproduce
1. Launch the local host http://localhost:4200/  
2. Navigate to the Log in section
3. See if "forgot password" has enough contrast with the bac
4. Test the contrast using any accessibility testing tool. For our case we used WAVE

### Expected behavior
The "forgot password" link should have enough contrast with the background. 

### Screenshots
![Uploading Screenshot from 2024-06-12 09-18-10.png…]()

**Comment by Angamanga** [comment]
@reuben Could you please check and suggest a new colour to use?

**Comment by Angamanga** [comment]
Closing and keeping in Linear https://linear.app/ushahidi/issue/USH-1432/[accessibility]-color-contrast-in-log-inforgot-password-link
